src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2131,7 +2131,7 @@
     <!-- CloneMethod may be null in error scenarios-->
     <Field Name="CloneMethod" Type="MethodSymbol?" />
     <!-- Members and expressions passed as arguments to the With expression. -->
-    <Field Name="Arguments" Type="ImmutableArray&lt;(Symbol? Member, BoundExpression Expression)&gt;" />
+    <Field Name="InitializerExpression" Type="BoundObjectInitializerExpressionBase" />
```

Comment:
Why was the type made `BoundObjectInitializerExpressionBase` instead of `BoundObjectInitializerExpression`? Is it just to make it easier to call `BindInitializerExpression`?

Reply:
No particular reason, I was just leaving space for potentially creating a `BoundWithInitializerExpression` in the future, if we want it.